app/models/user.rb

```diff
@@ -1,8 +1,9 @@
+require './lib/auth0_api'
+
 class User < ApplicationRecord
   has_many :memberships, dependent: :destroy
   has_many :suppliers, through: :memberships
   validates :email, presence: true, uniqueness: true
```

Comment:
Now we're allowing admins to create accounts, should we validate the format of the email address?

Reply:
Hmm, probably